index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
+    <header>
+        <div id="logo">
+            <img src="/media/My Logo.png">
+        </div>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
```

Comment:
Please make sure your color matches for the hamburger menu. Check the color difference below.
<img width="172" alt="Screenshot 2023-01-18 at 11 27 17 PM" src="https://user-images.githubusercontent.com/25887253/213287651-137e2e60-3e2e-4041-a03d-96fd985b2e7e.png">
<img width="172" alt="Screenshot 2023-01-18 at 11 27 34 PM" src="https://user-images.githubusercontent.com/25887253/213287656-0b79e07c-4821-441a-a8a8-cce741d7f809.png">